test/common/3scale_crudl.go

```diff
@@ -28,6 +28,9 @@ func Test3ScaleCrudlPermissions(t *testing.T, ctx *TestingContext) {
 
 	// Get the fuse host url from the rhmi status
 	host := rhmi.Status.Stages[v1alpha1.ProductsStage].Products[v1alpha1.Product3Scale].Host
+	if host == "" {
```

Comment:
hmm, I am wondering how Host could be empty, any ideas why is this happening @jjaferson ?

Reply:
`Test3ScaleSMTPConfig` patches the DC for the `system-app` [1] which modifies the installation status for 3scale to `in progress`, I don't think it would happen in PROW as the `Test3ScaleSMTPConfig` is the last test to run [2] but I spot that issue by running the functional test a few times

[1] https://github.com/integr8ly/integreatly-operator/blob/bbb544ff360c502719983badd181e284b04586bd/test/common/3scale_smtp.go#L294
[2] https://github.com/integr8ly/integreatly-operator/blob/master/test/common/tests.go#L53

Reply:
Oh, I see.
Not an ideal behavior, but it's probably minor enough...